src/test/app/NFToken_test.cpp

```diff
@@ -6798,6 +6798,320 @@ class NFToken0_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testFixNFTokenBuyerReserve(FeatureBitset features)
+    {
+        testcase("Test buyer reserve when accepting an offer");
+
+        using namespace test::jtx;
+
+        // Lambda that mints an NFT and then creates a sell offer
+        auto mintAndCreateSellOffer =
+            [](test::jtx::Env& env,
+               test::jtx::Account const& act,
+               STAmount const amt) -> std::tuple<uint256, uint256> {
+            // act mints a NFT
+            uint256 const nftId{token::getNextID(env, act, 0u, tfTransferable)};
+            env(token::mint(act, 0u), txflags(tfTransferable));
+            env.close();
+
+            // act makes an sell offer
+            uint256 const sellOfferIndex =
+                keylet::nftoffer(act, env.seq(act)).key;
+            env(token::createOffer(act, nftId, amt), txflags(tfSellNFToken));
+            env.close();
+
+            return {nftId, sellOfferIndex};
+        };
```

Comment:
Thanks for the suggestion. I think I would prefer removing the structured binding completely since it's indeed unnecessary to have the other field :)

Reply:
@scottschurr very insightful comment 👍 It's funny that you suggested the struct idea, because I believe that's how lambda's are implemented under the hood by the compiler :))